packages/cli/package.json

```diff
@@ -78,7 +78,8 @@
     "eslint": "^6.8.0",
     "eslint-plugin-jest": "^23.8.2",
     "jest": "^26.6.3",
-    "tmp-promise": "^2.0.2"
+    "tmp-promise": "^2.0.2",
+    "@ceramicnetwork/doctype-tile-handler": "^0.12.8"
```

Comment:
nitpick: can we keep these in lexographic order?  So this line should be between the `@babel` deps and the `@types` deps

Reply:
This reminds me of all the recommendations and punishments from 10 years ago when people tried to enforce code style by human labor. There were no prettier or gofmt. State of the art IDE was effectively vim. Seems like a wasted effort here, honestly.

Reply:
Hey, vim is great! 😅

Reply:
vim is awesome indeed.